examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -54,12 +54,16 @@ const ᶠcurlᵥ = Operators.CurlC2F(
     bottom = Operators.SetCurl(Geometry.Contravariant12Vector(FT(0), FT(0))),
     top = Operators.SetCurl(Geometry.Contravariant12Vector(FT(0), FT(0))),
 )
-const ᶜFC = Operators.FluxCorrectionC2C(
-    bottom = Operators.Extrapolate(),
-    top = Operators.Extrapolate(),
+const ᶠupwind1 = Operators.UpwindBiasedProductC2F()
+const ᶠupwind3 = Operators.Upwind3rdOrderBiasedProductC2F(
+    bottom = Operators.FirstOrderOneSided(),
+    top = Operators.FirstOrderOneSided(),
```

Comment:
Tapio mentioned a few times that we want to use the `ThirdOrderOneSided` BCs for the `Upwind3rdOrderBiasedProductC2F` operator, for accuracy and convergence (you can see the unit tests for this operator where it's clear that the `ThirdOrderOneSided` BC outperforms the `FirstOrderOneSided` one). In the FCT operator though, we decided to use the lower-order BCs as they reduced some wiggles

Reply:
Gotcha, I'll switch to ThirdOrderOneSided for that one.

Reply:
So please change this

Reply:
https://github.com/CliMA/ClimaAtmos.jl/pull/790/commits/e01104c761ba67edab26e1eb00d332cafd5d47c3